### ISSUE TITLE
Consider using a date instead of unix timestamp

### ISSUE THREAD

**Issue by asabil** [mbucc/shmig#32]
A timestamp in the form of YYYYMMDDHHMMSS is generally easier to relate to than unix timestamps.

**Comment by mbucc** [comment]
I usually only care about the ordering and the time stamp is added to the file itself.

Also, epoch seconds include timezone (they are always in GMT) which is a win.

Thanks for your input, but I'm going to close this.

**Comment by kael-shipman** [comment]
Hey all,

As the author of a similar issue, #42, I just wanted to add my two cents to this.

In my opinion, the point of implementing this as a _config item_ is so we can both have our way. If we can figure out the ordering issues that I ran into implementing this in the database (which to me are fairly minor), then adding this feature really won't affect you at all :).

Sure, we don't just want to implement any old feature that gets suggested, but this seems like one of those personality things where half the people are going to want to do it one way and the other half are going to want to do it the other way, and it's easy to give them both their way.

I've already started implemented this (see my [human-timestamps branch](https://github.com/mbucc/shmig/compare/master...cfxmarkets:human-timestamps)), so it would be great if we could figure out how to get it into `master` so everyone can be happy :D.
